apps/storybook.namekit.io/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,694 @@
+      currency: Currency.Usdc,
+      value: 1n,
+    },
+  },
+};
+export const UsdcMaxDisplayPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Usdc,
+      value: 1000000000000000000n,
+    },
+  },
+};
+export const DaiMinDisplayPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Dai,
+      value: 1n,
+    },
+  },
+};
+export const DaiMaxDisplayPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Dai,
+      value: 10000000000000000000000000n,
+    },
+  },
+};
+export const WethMinDisplayPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Weth,
+      value: 1n,
+    },
+  },
+};
+export const WethMaxDisplayPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Weth,
+      value: 1000000000000000000000000n,
+    },
+  },
+};
+export const UsdPriceAndSymbolExample: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    symbol: (
+      <p>
+        You can define your custom symbology by informing a React.ReactNode to
+        the "symbol" prop!
+      </p>
+    ),
+  },
+};
+export const UsdPriceWithSymbol: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    symbol: (
+      <CurrencySymbol
+        describeCurrencyInTooltip={true}
+        currency={Currency.Usd}
+      />
+    ),
+    symbolPosition: CurrencySymbolPosition.Left,
+  },
+};
+export const UsdPriceWithAcronym: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    symbol: (
+      <span>
+        {getCurrencySymbology(Currency.Usd, CurrencySymbology.Acronym)}
+      </span>
+    ),
+    symbolPosition: CurrencySymbolPosition.Left,
+  },
+};
+export const UsdPriceWithSymbolOnRight: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    symbol: (
+      <CurrencySymbol
+        describeCurrencyInTooltip={true}
+        currency={Currency.Usd}
+      />
+    ),
+    symbolPosition: CurrencySymbolPosition.Right,
+  },
+};
+export const UsdPriceWithAcronymOnRight: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    symbol: (
+      <span>
+        {getCurrencySymbology(Currency.Usd, CurrencySymbology.Acronym)}
+      </span>
+    ),
+    symbolPosition: CurrencySymbolPosition.Right,
+  },
+};
+export const EthPriceWithSymbol: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    symbol: (
+      <CurrencySymbol
+        describeCurrencyInTooltip={true}
+        currency={Currency.Eth}
+      />
+    ),
+    symbolPosition: CurrencySymbolPosition.Left,
+  },
+};
+export const EthPriceWithAcronym: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    symbol: (
+      <span>
+        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
```

Comment:
Suggest updating the `CurrencySymbol` UI component so that it has a new prop: `symbology`. This can take in a value of type `CurrencySymbology`.

You should then move these acronym and other custom symbology use cases to Storybook stories for `CurrencySymbol`.

In other words, "ETH" is also a `CurrencySymbol`! The `CurrencySymbol` UI component should have the logic inside of itself to handle this case.

When we write the JSX for an app UI, we want to make the markup nicely self documenting. Ex: All these cases are just `CurrencySymbol` with different symbology.

Reply:
Sounfs good idea!